message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1502,18 +1502,22 @@
      <field type="uint8_t" name="health">EFI Health status</field>
      <field type="float"   name="ecu_index">ECU Index</field>
      <field type="float"   name="rpm">RPM</field>
-     <field type="float"   name="fuel_consumed" units="g">Fuel Consumed (grams)</field>
-     <field type="float"   name="fuel_flow" units="g/min">Fuel Flow Rate (g/min)</field>
-     <field type="float"   name="engine_load" units="%">Engine Load (%)</field>
-     <field type="float"   name="throttle_position" units="%">Throttle Position (%)</field>
-     <field type="float"   name="spark_dwell_time" units="ms">Spark Dwell Time (ms)</field>
-     <field type="float"   name="barometric_pressure" units="kPa">Barometric Pressure (kPa)</field>
-     <field type="float"   name="intake_manifold_pressure" units="kPa">Intake Manifold Pressure (kPa)(</field>
-     <field type="float"   name="intake_manifold_temperature" units="degC">Intake Manifold Temperature (degC)</field>
-     <field type="float"   name="cylinder_head_temperature" units="degC">cylinder_head_temperature (degC)</field>
-     <field type="float"   name="ignition_timing" units="deg">Ignition timing for cylinder i (Crank Angle degrees)</field>
-     <field type="float"   name="injection_time" units="ms">Injection time for injector i (ms)</field>
-   </message>
+     <field type="float"   name="fuel_consumed" units="g">Fuel Consumed</field>
+     <field type="float"   name="fuel_flow" units="g/min">Fuel Flow Rate</field>
+     <field type="float"   name="engine_load" units="%">Engine Load</field>
+     <field type="float"   name="throttle_position" units="%">Throttle Position</field>
+     <field type="float"   name="spark_dwell_time" units="ms">Spark Dwell Time</field>
+     <field type="float"   name="barometric_pressure" units="kPa">Barometric Pressure</field>
+     <field type="float"   name="intake_manifold_pressure" units="kPa">Intake Manifold Pressure(</field>
+     <field type="float"   name="intake_manifold_temperature" units="degC">Intake Manifold Temperature</field>
+     <field type="float"   name="cylinder_head_temperature" units="degC">cylinder_head_temperature</field>
+     <field type="float"   name="ignition_timing" units="deg">Ignition timing (Crank Angle degrees)</field>
+     <field type="float"   name="injection_time" units="ms">Injection time</field>
+     <extensions/>
```

Comment:
Should we really bother making these extensions?  How many users of this message are there?